packages/material/src/controls/MaterialInputControl.tsx

```diff
@@ -36,7 +36,7 @@ import { Hidden, InputLabel } from '@material-ui/core';
 import { FormControl, FormHelperText } from '@material-ui/core';
 import merge from 'lodash/merge';
 
-interface WithInput {
+export interface WithInput {
   input: any;
```

Comment:
Should we type this?

Reply:
In general yes, but I would not change this now to avoid typing problems in user code.